website/src/components/header/Header.jsx

```diff
@@ -135,7 +128,15 @@ const Header = () => {
             <figure className="lineMenuBottom"></figure>
           </figure>
         )}
+        
       </div>
+      <button className="buttonMode" onClick={handleButtonClick}>
```

Comment:
I think it would be better if the mode switch button were to the left of the hamburger menu on mobile. Otherwise, I really like the button!

Reply:
I have made these changes.